fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeTypes.java

```diff
@@ -0,0 +1,27 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.privilege;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum PrivilegeTypes {
+    TABLE(TableActions.values());
+
+    private final List<String> validActions;
+    PrivilegeTypes(Object[] validObject) {
+        this.validActions = Stream.of(validObject).map(o -> o.toString()).collect(Collectors.toList());
+    }
+
+    public List<String> getValidActions() {
+        return validActions;
+    }
+
+    /**
+     * Below defines all validate actions of a certain type
+     */
+    public enum TableActions {
+        SELECT;
```

Comment:
There should be other actions to add for TABLE type object?

Reply:
Yes, but not in this PR. As I wrote in the description of this PR, only select action on table level is implemented.

Reply:
ok